extensions/flyway/deployment/src/main/java/io/quarkus/flyway/FlywayProcessor.java

```diff
@@ -12,14 +12,7 @@
 import java.nio.file.Files;
 import java.nio.file.Path;
 import java.nio.file.Paths;
-import java.util.ArrayList;
-import java.util.Collection;
-import java.util.Enumeration;
-import java.util.HashMap;
-import java.util.LinkedHashSet;
-import java.util.List;
-import java.util.Map;
-import java.util.Set;
+import java.util.*;
```

Comment:
Oh my bad, I did not see this star import. We usually do not allow them in the codebase. Sorry I should have spotted it. 

Can you please deal import individual classes and get rid of it (making sure that it is into one commit) ? Thank you

Reply:
It doesn't look solved to me?